.travis.yml

```diff
@@ -2,12 +2,11 @@ language: php
 sudo: false
 
 php:
-  - 7.0
   - 7.1
   - nightly
 
 before_script:
-  - composer install --prefer-dist -o
+  - composer install -o
```

Comment:
Don't use `-o` for tests, as it will cause the test suite to succeed even if there is a misnamed class